python-checks/src/main/java/org/sonar/python/checks/IgnoredSystemExitCheck.java

```diff
@@ -154,7 +165,14 @@ private static boolean handleCaughtException(SubscriptionContext ctx, Expression
     }
 
     if (SYSTEM_EXIT_EXCEPTION_NAME.equals(caughtExceptionName)) {
-      ctx.addIssue(caughtException, MESSAGE_NOT_RERAISED_CAUGHT_EXCEPTION);
+      var issue = (IssueWithQuickFix) ctx.addIssue(caughtException, MESSAGE_NOT_RERAISED_CAUGHT_EXCEPTION);
+      if (visitor.passStatement != null) {
+        var quickFix = PythonQuickFix.newQuickFix(NOT_RAISED_CAUGHT_EXCEPTION_QUICK_FIX_MESSAGE)
+          .addTextEdit(PythonTextEdit.replace(visitor.passStatement, "raise"))
+          .build();
+        issue.addQuickFix(quickFix);
```

Comment:
I think in all cases where we raise an issue, a `SystemExit` is ignored. Thus, I think it's valid to suggest that quick fix for the other scenarios where we raise an issue.

Reply:
done